zhaquirks/tuya/ts0201.py

```diff
@@ -0,0 +1,219 @@
+"""Tuya TS201 temperature, humidity and optional illumination sensors."""
+
+from zigpy.profiles import zha
+from zigpy.profiles.zha import DeviceType
+from zigpy.quirks import CustomCluster, CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import Basic, Identify, Ota, PowerConfiguration, Time
+from zigpy.zcl.clusters.measurement import (
+    IlluminanceMeasurement,
+    RelativeHumidity,
+    TemperatureMeasurement,
+)
+from zigpy.zdo.types import NodeDescriptor
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    NODE_DESCRIPTOR,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+
+class ValueAlarm(t.enum8):
+    """Temperature and humidity alarm values."""
+
+    ALARM_OFF = 0x02
+    MAX_ALARM_ON = 0x01
+    MIN_ALARM_ON = 0x00
+
+
+class TuyaTemperatureHumidityAlarmCluster(CustomCluster):
+    """Neo temperature and humidity alarm cluster (0xE002)."""
```

Comment:
Also the comment can be changed to """Tuya temperature and ...

Reply:
Agree, have done it.